src/pybind/mgr/ansible/module.py

```diff
@@ -346,6 +412,35 @@ def remove_osds(self, osd_ids):
 
         return ansible_operation
 
+    def add_host(self, host, role):
+        # type: (str, str) -> WriteCompletion
```

Comment:
`WriteCompletion` is not resolvable in this context. if you don't use an editor that supports type deduction, I'd leave it out.

Reply:
I think that is useful to indicate the way to follow in each of the orchestrators.  For me it was useful to know what type of object should return this method.

Reply:
yes! this is also part of the documentation, correct. what about `S/WriteCompletion/orchestrator.WriteCompleteion/`?